fixes.inc

```diff
@@ -8533,29 +8531,20 @@ native BAD_GetPlayerMenu(playerid) = GetPlayerMenu;
 				{
 						new
 							model = GetPlayerVehicleSeat(playerid);
-						if (model != 128)
+						if (model == 0)
 						{
-							if (model == 0)
-							{
-								model = GetVehicleModel(GetPlayerVehicleID(playerid)) - 400;
-								if (_FIXES_NO_RANGE(model, 0, 211 + 1) || FIXES_gscMaxPassengers[model >>> 3] >>> ((model & 7) << 2) & 0xF == 15)
-								{
-									// In an invalid vehicle (one you can't drive).
-									FIXES_BlockUpdate(playerid);
-								}
-							}
-							else
+							model = GetVehicleModel(GetPlayerVehicleID(playerid)) - 400;
+							if (_FIXES_NO_RANGE(model, 0, 211 + 1) || FIXES_gscMaxPassengers[model >>> 3] >>> ((model & 7) << 2) & 0xF == 15)
 							{
-								// They are a driver, but not in the driver's seat.
+								// In an invalid vehicle (one you can't drive).
 								FIXES_BlockUpdate(playerid);
 							}
 						}
-					#if FIX_HydraSniper
```

Comment:
This part:
```
#if FIX_HydraSniper
	else
	{
		model = GetVehicleModel(GetPlayerVehicleID(playerid));
	}
#endif
```
was cut at all as I don't understand why we need to GetVehicleModel and then just end that code without doing any actions with model variable.

Reply:
It's used here:

https://github.com/Open-GTO/sa-mp-fixes/blob/f2700884614c8304b063c940fe4ab141c3328abe/fixes.inc#L8567

Reply:
But there is a '}' right after it so it sholud be the different bodies of code.